src/roblox/init.lua

```diff
@@ -1487,6 +1500,18 @@ local function bitBuffer(stream)
         end
     end
 
+    local function readDateTime()
+        return DateTime.fromUniversalTime(
```

Comment:
In general I like to have assertions to check if there's enough space left in the buffer to read the datatype.

Reply:
(I'll override the Selene wormflow since I know DateTime exists. Will make sure it ends up in the next release.)

Reply:
So do I do `assert(pointer + 50 <= bitCount, "BitBuffer.readDateTime cannot read past the end of the stream")`?

Reply:
Should be good now.